chat/models.py

```diff
@@ -116,11 +116,11 @@ def send_message(self, msg):
             service = self.get_service()
             service.send_message(self.text_channel_id, msg)
 
-    def handle_tag_added(self, puzzle, tag):
-        self.get_service().handle_tag_added(self, puzzle, tag)
+    def handle_tag_added(self, puzzle, tag_name):
+        self.get_service().handle_tag_added(puzzle, tag_name)
```

Comment:
What's the reason for having handle_tag_added jump around from tasks to the chat model back to discord_chat_service.py?

Reply:
IMO our initial plan was not to expose the service on any views (or tasks). This is WAI IMO.

Reply:
Gotcha, makes sense